test/appium/views/about_view.py

```diff
@@ -0,0 +1,28 @@
+from views.base_element import BaseButton, BaseText
+from views.base_view import BaseView
+
+
+class PrivacyPolicyButton(BaseButton):
+    def __init__(self, driver):
+        super(PrivacyPolicyButton, self).__init__(driver)
+        self.locator = self.Locator.xpath_selector(
+            '//android.view.ViewGroup[@content-desc="privacy-policy"]/android.view.ViewGroup')
```

Comment:
please, use more general xpath for all added elements
e.g. //*[@content-desc="privacy-policy"]